### PR TITLE
fix(restclient): allow subsequent in path parameters in url

### PR DESCRIPTION
**Pull request by pevisscher**

### CODE REVIEW

**Comment by pevisscher** [comment]
Current dependencies on/for this PR:
* main
  * **PR #27** <a href="https://app.graphite.dev/github/pr/skyleague/therefore/27" target="_blank">Graphite</a>  👈
    * **PR #28** <a href="https://app.graphite.dev/github/pr/skyleague/therefore/28" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/skyleague/therefore/27?utm_source=stack-comment).

**Review by loredanamoanga — APPROVED** [review]

**Comment by pevisscher** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/skyleague/therefore/27?utm_source=stack-comment).


Job Reference: zLEF4DukGo7omGVAGyG2

**Pull request merged**
